hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailPlugin.java

```diff
@@ -302,9 +302,9 @@ protected Message createMimeMessage(ActionMessage msg) throws Exception {
             if (alert.getStatus().equals(Status.OPEN)) {
                 email.setSentDate(new Date(alert.getCtime()));
             } else if (alert.getStatus().equals(Status.ACKNOWLEDGED)) {
-                email.setSentDate(new Date(alert.getAckTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             } else {
-                email.setSentDate(new Date(alert.getResolvedTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             }
```

Comment:
For REST users they will deal with the JSON representation then they will deal just with the lifecycle list.
But for Plugins developers who have access to Java Alerts API these methods can be useful so I will include them on the PR.